qiskit_experiments/analysis/curve_analysis.py

```diff
@@ -71,6 +72,8 @@ class CurveAnalysis(BaseAnalysis):
             Each series definition is SeriesDef element, that may be initialized with:
 
                 fit_func: The function to which the data will be fit.
+                monotonicity: a list of Booleans, whose length is the number of parameters.
```

Comment:
Why is this monotonicity? It only covers increasing functions, not decreasing functions; are we assuming each function is always either increasing or decreasing?

Reply:
We can compute a confidence interval only if the function is either increasing or decreasing for each parameter, all these 2^{number of parameters} cases are covered here. Otherwise it is impossible to compute a confidence interval, and in this case `monotonicity` has to be `None`.

Reply:
Thanks. Still not sure about the implicit naming convention (that `True` means increasing and `False` means decreasing).

Reply:
Maybe change from `monotonicity` to `is_increasing`. Or keep the name `monotonicity`, and expect either +1 or -1 in each list element. What do you think?